### DISCUSSION TITLE
CF2021 and Masa

### DISCUSSION THREAD

**Discussion by brimard** [MasaCMS/MasaCMS#53] [Q&A] [answered]
Is anyone running Masa on CF2021?

**Comment by jimblesphere** [comment, marked as answer]
I am running Masa 7.3 on ColdFusion 2021. I had to make several changes to
the core code to make it work under both CF2016 and CF2021. I submitted a
pull request to incorporate these changes into the repository.

**Reply by brimard** [reply]
Thanks, still evaluating our update options from Mura, and with it, which CFML engine.
